### ISSUE TITLE
Missing hdf_50.h5 in charter 6

### ISSUE THREAD

**Issue by williamtong0228** [PacktPublishing/Hands-On-Deep-Learning-for-Finance#17]
Hi, recently I bought your book but chapter 6's examples are hard to follow step by step on Jupyter Notebook. 

The examples only generated simpleclass_train_50.h5, but the model only has two keys. Don't know if it refers to the same file. 
X_tr, X_vld, y_tr, y_vld, list_ch_train = get_tf_train_data(traindtfile) 
this line of code says that it cannot find the object if I rename simpleclass_train_50.h5. And I am not sure what data I should use to add keywords to the .h5 file.

**Comment by abhandari73** [comment]
Hi William,

Surely will help you get it running.

Can you tell me where did you purchase the book from?


Kind Regards,
Arjun Bhandari
+97150 780 5672

**Comment by williamtong0228** [comment]
Hi Arjun,
I bought it from Apple Books a couple days ago. I think you could update the book versions in the app stores, or add some hints in the repositories. Thank you.
William

**Comment by JosephA-packt** [comment]
The book has been discontinued @williamtong0228
